doc/manuals/devel/mongoBackend.md

```diff
@@ -361,12 +361,9 @@ _MB-12: mongoUpdateSubscription_
 
 * `mongoUpdateSubscription()` is invoked from a service routine (step 1). This can be from either `patchSubscription()` (which resides in `lib/serviceRoutinesV2/patchSubscription.cpp`) or `mongoUpdateContextSubscription()` (which resides in `lib/mongoBackend/mongoUpdateContextSubscription.cpp`).
 * Depending on `-reqMutexPolicy`, the request semaphore may be taken (write mode) (step 2). See [this document for details](semaphores.md#mongo-request-semaphore). 
-* The subscription to be updated is retrieved from the database using `collectionFindOne()` in the `connectionOperations` module (steps 3 and 4).
-* If the subscription cache is enabled (i.e. `noCache` set to `false`) the subscription cache object is also retrieved from the subscription cache using `subCacheItemLoopkup()` in the `cache` module (step 5).
-* The BSON object of the final subscription is built, based on the BSON object of the original subscription, using different `set*()` functions similar to the ones in the create subscription case (`setExpiration()`, `setHttpInfo()`, etc.). (step 6).
-* The BSON object corresponding to the updated subscription is updated in the database using `collectionUpdate()` in the `connectionOperations` module (steps 7 and 8).
-* In case the subscription cache is enabled  (i.e. `noCache` set to `false`) the new subscription is updated in the subscription cache (step 9). `updateInCache()` uses the subscription cache semaphore internally.
-* If the request semaphore was taken in step 2, then it is released before returning (step 10).
+* An update request is build using MongoDB `$set`/`$unset` operators, then used to update the subscription in DB using `colletionFindAndModify()` in the `connectionOperations` module (steps 3 and 4).

```

Comment:
I think this sentence has to be rephrased, it is not clear what is done in this step
```suggestion
The subscription is updated in DB by a request using MongoDB `$set`/`$unset` operators. This operation is done in the function `colletionFindAndModify()` in the `connectionOperations` module (steps 3 and 4).

```

Reply:
```suggestion
* The subscription is updated in DB using MongoDB `$set`/`$unset` operators. This operation is done in the function `colletionFindAndModify()` in the `connectionOperations` module (steps 3 and 4).
```

Reply:
Fixed in b4ee08b